packages/core/src/extensions/extension-loader/file-system-provisioner-store/get-hash.injectable.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+import { SHA256 } from "crypto-js";
+
+const getHashInjectable = getInjectable({
+  id: "get-hash",
+
+  instantiate: () => (text: string) => SHA256(text).toString(),
```

Comment:
Should we add `causesSideEffects: true` since used external imported package?

Reply:
In don't think that is necessary because a hash should be stable given its input